fotoapparat/src/main/java/io/fotoapparat/parameter/Zoom.kt

```diff
@@ -0,0 +1,12 @@
+package io.fotoapparat.parameter
+
+/**
+ * Zoom modes which camera can use.
+ */
+sealed class Zoom {
+
+    object FixedZoom : Zoom()
+
+    data class VariableZoom(val minZoom: Int, val maxZoom: Int) : Zoom()
```

Comment:
missing doc e.g. : `The camera can only support a variable zoom level between (and including) [minZoom] and [maxZoom] values`

Reply:
Ok, added.